joomla/joomla-cms
# Update npm dependencies

### Summary of Changes
This is the result of a `npm update` to update our dependencies to their current versions as supported by our PHP minimum standard.

@brianteeman I cannot update the `skipto` plugin to anything higher than 4.1.3 because the newer versions are missing the `compiled` folder. Is that something you can look into?

The output of `npm outdated` **before**
```
Package                        Current  Wanted  Latest  Location                                    Depended by
@babel/core                     7.16.7  7.18.2  7.18.2  node_modules/@babel/core                    joomlacms.test
@babel/preset-env               7.16.7  7.18.2  7.18.2  node_modules/@babel/preset-env              joomlacms.test
@fortawesome/fontawesome-free   5.15.4  5.15.4   6.1.1  node_modules/@fortawesome/fontawesome-free  joomlacms.test
@popperjs/core                  2.11.0  2.11.5  2.11.5  node_modules/@popperjs/core                 joomlacms.test
@rollup/plugin-babel             5.3.0   5.3.1   5.3.1  node_modules/@rollup/plugin-babel           joomlacms.test
@rollup/plugin-commonjs         21.0.1  21.1.0  22.0.0  node_modules/@rollup/plugin-commonjs        joomlacms.test
@rollup/plugin-node-resolve     13.1.2  13.3.0  13.3.0  node_modules/@rollup/plugin-node-resolve    joomlacms.test
@rollup/plugin-replace           3.0.1   3.1.0   4.0.0  node_modules/@rollup/plugin-replace         joomlacms.test
@vue/compiler-sfc               3.2.26  3.2.37  3.2.37  node_modules/@vue/compiler-sfc              joomlacms.test
accessibility                   3.0.15  3.0.16  3.0.16  node_modules/accessibility                  joomlacms.test
autoprefixer                    10.4.1  10.4.7  10.4.7  node_modules/autoprefixer                   joomlacms.test
choices.js                       9.1.0   9.1.0  10.1.0  node_modules/choices.js                     joomlacms.test
chokidar                         3.5.2   3.5.3   3.5.3  node_modules/chokidar                       joomlacms.test
codemirror                      5.65.2  5.65.5   6.0.0  node_modules/codemirror                     joomlacms.test
commander                        8.3.0   8.3.0   9.3.0  node_modules/commander                      joomlacms.test
core-js                         3.20.1  3.22.8  3.22.8  node_modules/core-js                        joomlacms.test
cssnano                         5.0.14  5.1.11  5.1.11  node_modules/cssnano                        joomlacms.test
diff                             5.0.0   5.1.0   5.1.0  node_modules/diff                           joomlacms.test
eslint                           8.5.0  8.17.0  8.17.0  node_modules/eslint                         joomlacms.test
eslint-plugin-import            2.25.3  2.26.0  2.26.0  node_modules/eslint-plugin-import           joomlacms.test
eslint-plugin-vue                8.2.0   8.7.1   9.1.1  node_modules/eslint-plugin-vue              joomlacms.test
fs-extra                        10.0.0  10.1.0  10.1.0  node_modules/fs-extra                       joomlacms.test
ini                              2.0.0   2.0.0   3.0.0  node_modules/ini                            joomlacms.test
jasmine-core                    3.10.1  3.99.1   4.2.0  node_modules/jasmine-core                   joomlacms.test
jquery-migrate                   3.3.2   3.4.0   3.4.0  node_modules/jquery-migrate                 joomlacms.test
karma                           6.3.17  6.3.20  6.3.20  node_modules/karma                          joomlacms.test
karma-coverage                   2.1.0   2.2.0   2.2.0  node_modules/karma-coverage                 joomlacms.test
karma-jasmine                    4.0.1   4.0.2   5.0.1  node_modules/karma-jasmine                  joomlacms.test
karma-verbose-reporter           0.0.6   0.0.6   0.0.8  node_modules/karma-verbose-reporter         joomlacms.test
mediaelement                     5.0.4   5.0.5   5.0.5  node_modules/mediaelement                   joomlacms.test
postcss                          8.4.5  8.4.14  8.4.14  node_modules/postcss                        joomlacms.test
postcss-scss                     4.0.2   4.0.4   4.0.4  node_modules/postcss-scss                   joomlacms.test
rollup                          2.62.0  2.75.6  2.75.6  node_modules/rollup                         joomlacms.test
sass                            1.49.9  1.52.3  1.52.3  node_modules/sass                           joomlacms.test
selenium-standalone              8.0.8   8.1.4   8.1.4  node_modules/selenium-standalone            joomlacms.test
skipto                           4.1.3   4.1.6   4.1.6  node_modules/skipto                         joomlacms.test
stylelint                       14.2.0  14.9.1  14.9.1  node_modules/stylelint                      joomlacms.test
stylelint-config-standard       24.0.0  24.0.0  26.0.0  node_modules/stylelint-config-standard      joomlacms.test
stylelint-scss                   4.1.0   4.2.0   4.2.0  node_modules/stylelint-scss                 joomlacms.test
terser                           5.7.0  5.14.1  5.14.1  node_modules/terser                         joomlacms.test
tinymce                         5.10.3  5.10.5   6.0.3  node_modules/tinymce                        joomlacms.test
vue                             3.2.26  3.2.26  3.2.37  node_modules/vue                            joomlacms.tes
```
The output of `npm outdated` **after**
```
Package                        Current  Wanted  Latest  Location                                    Depended by
@fortawesome/fontawesome-free   5.15.4  5.15.4   6.1.1  node_modules/@fortawesome/fontawesome-free  joomlacms.test
@rollup/plugin-commonjs         21.1.0  21.1.0  22.0.0  node_modules/@rollup/plugin-commonjs        joomlacms.test
@rollup/plugin-replace           3.1.0   3.1.0   4.0.0  node_modules/@rollup/plugin-replace         joomlacms.test
choices.js                       9.1.0   9.1.0  10.1.0  node_modules/choices.js                     joomlacms.test
codemirror                      5.65.5  5.65.5   6.0.0  node_modules/codemirror                     joomlacms.test
commander                        8.3.0   8.3.0   9.3.0  node_modules/commander                      joomlacms.test
eslint-plugin-vue                8.7.1   8.7.1   9.1.1  node_modules/eslint-plugin-vue              joomlacms.test
ini                              2.0.0   2.0.0   3.0.0  node_modules/ini                            joomlacms.test
jasmine-core                    3.99.1  3.99.1   4.2.0  node_modules/jasmine-core                   joomlacms.test
karma-jasmine                    4.0.2   4.0.2   5.0.1  node_modules/karma-jasmine                  joomlacms.test
karma-verbose-reporter           0.0.6   0.0.6   0.0.8  node_modules/karma-verbose-reporter         joomlacms.test
skipto                           4.1.3   4.1.3   4.1.6  node_modules/skipto                         joomlacms.test
stylelint-config-standard       24.0.0  24.0.0  26.0.0  node_modules/stylelint-config-standard      joomlacms.test
tinymce                         5.10.5  5.10.5   6.0.3  node_modules/tinymce                        joomlacms.test
vue                             3.2.26  3.2.26  3.2.37  node_modules/vue                            joomlacms.test
```

### Testing Instructions
1. Checkout this branch
2. Run `npm ci` in the root of your site
3. Check if the frontend and backend work as expected.

### Actual result BEFORE applying this Pull Request
The site works as expected.


### Expected result AFTER applying this Pull Request
The site works as expected.


### Documentation Changes Required
None
